spinnman/transceiver/base_transceiver.py

```diff
@@ -1146,6 +1146,7 @@ def set_ip_tag(self, ip_tag: IPTag, use_sender: bool = False):
         for connection in connections:
             # Convert the host string
             host_string = ip_tag.ip_address
+            assert host_string is not None
```

Comment:
Just a thought... if host_string can be None, why don't we just add that in to the set of things that makes it local_ip_address?

Reply:
no longer needed. Reverted.